forge-gui/res/cardsfolder/upcoming/water_weird.txt

```diff
@@ -0,0 +1,10 @@
+Name:Water Weird
+ManaCost:3 U
+Types:Creature Elemental Weird
+PT:3/4
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Opponent | CombatDamage$ True | TriggerZones$ Battlefield | Execute$ TrigReveal | TriggerDescription$ Whenever CARDNAME deals combat damage to a player, put a +1/+1 counter on CARDNAME if the top card of your library is a nonland card. Otherwise, you may mill a card.
+SVar:TrigReveal:DB$ PeekAndReveal | PeekAmount$ 1 | NoPeek$ True | NoReveal$ True | RememberPeeked$ True | SubAbility$ DBCounter
+SVar:DBCounter:DB$ PutCounter | ConditionDefined$ Remembered | ConditionPresent$ Card.nonLand | ConditionCompare$ EQ1 | CounterType$ P1P1 | CounterNum$ 1 | Execute$ TrigMill
+SVar:TrigMill:DB$ Mill | NumCards$ 1 | Defined$ You | ConditionDefined$ Remembered | ConditionNotPresent$ Card | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+Oracle:Whenever Water Weird deals combat damage to a player, put a +1/+1 counter on Water Weird if the top card of your library is a nonland card. Otherwise, you may mill a card.

```

Comment:
This should be the rebalanced script:
![image](https://user-images.githubusercontent.com/18243520/184670091-cafa8ef7-2d94-4ce1-9e53-c63dde5c1f7d.png)

Reply:
For this card yes as it's an Alchemy only card and there is only one version on Arena.